source/index.js

```diff
@@ -2,15 +2,22 @@ var Pixi = require("pixi.js")
 var Afloop = require("afloop")
 var Keyb = require("keyb")
 
-var WIDTH = 230, HEIGHT = 130
+import Player from "./player.js"
```

Comment:
Yep, works just dandy. Also more familiar to someone like me coming from Java and Python.